docs/en/administrator-guide/config/fe_config.md

```diff
@@ -366,20 +366,16 @@ MasterOnly：true
 
 Whether to enable spark load temporarily, it is not enabled by default
 
-### enable_strict_storage_medium_check
+### disable_storage_medium_check
```

Comment:
It's not a good idea to change a config key for compatiable reason. Suppose a user set `enable_strict_storage_medium_check` to true, after he/she upgraded, the old config has no effect and would complain.

Reply:
Actually, now, this config is a bad design, so I forcibly modified this configuration. For users, it can be divided into two categories, one is concerned about the storage media, the other is not concerned. if enable_strict_storage_medium_check is true, the behavior is same as that disable_storage_medium_check is false. If  enable_strict_storage_medium_check is false, it would cause continuous migration task failure which is a bug that should be fixed.

Reply:
It's not necessary to change the config name when you fix the bug.

Reply:
enable_strict_storage_medium_check is not same as  enable_storage_medium_check, if we don't remove this config, it is still a bug, you could see the comment about this config. and enable_strict_storage_medium_check -> enable_storage_medium_check or disable_storage_medium_check, I think it is all right. for older user, new config would let user know the wrong usage for the storage medium, and for new user, for new user, we have already correct usage

Reply:
Does the bug exits in all cases?

Reply:
further more, if it's a bug, better to describe it on PR title or description, or lable.

Reply:
> Does the bug exist in all cases?

for enable_strict_storage_medium_check=false, on condition that most user cannot be aware this problem, but it may cause unstabitily of fe.

Reply:
> BTW, if it's a bug, better to describe it on PR title or description, or add a `bug` lable.

ok